### PR TITLE
Deleted invalid policies for Android

### LINKED ISSUE #309: Remove these policies for Android devices
These policies cannot be applied to the Android agent please remove:

- disableFmRadio
- disableVoiceMail
- disableCallAutoAnswer
- disableVoiceDictation
- disableUsbOnTheGo

these policies don't have any API or script to control on Android at this moment

### PR DESCRIPTION
**Pull request by DIOHz0r**
### Changes description

Some policies created are not valid for Android devices.

### References

Closes #309

### CODE REVIEW

**Comment by btry** [comment]
Please completely remove the entries in the big array, and in  the upgrade code, add some requests to remove these entries.

File: install/upgrade/update_to_dev.php [hunk 1]
```diff
@@ -180,6 +180,7 @@ function plugin_flyvemdm_update_to_dev(Migration $migration) {
    $migration->addKey($table, 'plugin_flyvemdm_policycategories_id', 'plugin_flyvemdm_policycategories_id');
    // All policies exist for Android
    $migration->addPostQuery("UPDATE `$table` SET `is_android_policy` = '1'");
+   $migration->addPostQuery("DELETE FROM `$table` WHERE `symbol` IN ('disableFmRadio','disableVoiceMail','disableCallAutoAnswer','disableVoiceDictation','disableUsbOnTheGo')");
```
Comment (btry) [inline install/upgrade/update_to_dev.php]: We need to remove tasks using these policies too

Comment (DIOHz0r) [reply install/upgrade/update_to_dev.php]: do you mean the cron task, don't you?

Comment (btry) [reply install/upgrade/update_to_dev.php]: No, I mean the tables having a foreign key to policies.

All tables having a foreign key to policies miust be cleaned up. If I remember correctly, only glpi_plugin_flyvemdm_tasks has such key

Also, we must send MQTT messages to delete topics of these policies, if any.

File: install/upgrade/update_to_dev.php [hunk 2]
```diff
@@ -180,7 +180,30 @@ function plugin_flyvemdm_update_to_dev(Migration $migration) {
...
+      'INNER JOIN' => [
+         $tasksTable => ['FKEY' => [$tasksTable => 'plugin_flyvemdm_policies_id', $table => 'id']],
+      ],
+      'WHERE'      => ['symbol' => $policies],
+   ];
+   $result = $DB->request($request);
+   if (count($result) > 0) {
+      $mqttClient = PluginFlyvemdmMqttclient::getInstance();
+      foreach ($DB->request($result) as $row => $data) {
+         $mqttClient->publish($data['plugin_flyvemdm_fleets_id'] . "/Policy/" . $data['symbol'], null, 0, 1);
```
Comment (btry) [inline install/upgrade/update_to_dev.php]: There are some changes in the MQTT topics, and some others pending (leading slash)

Currently a topic has this structure

```/0/fleet/38/Policy/storageEncryption/Task/37```
- entity ID
- litteral **fleet**
- fleet ID
- litteral **Policy**
- symbol of a policy
- litteral **Task**
- task ID

Moreover, the leading slash in the topic is not yet removed (PR #315 issue #313).

You may pick some more columns in your request to g et the missing IDs and complete the MQTT topic.

**Pull request merged**